docs/python/tutorial-azure-functions.md

```diff
@@ -0,0 +1,348 @@
+            total = 0
+            for j in range(i, 0, -1):
+                total = (total * j) + (scale * arr[j])
+                arr[j] = total % ((j * 2) - 1)
+                total = total / ((j * 2) - 1)
+
+            output += "%04d" % (carry + (total / scale))
+            carry = total % scale
+
+        return output;
+
+    def main(req: func.HttpRequest) -> func.HttpResponse:
+        logging.info('DigitsOfPi HTTP trigger function processed a request.')
+
+        digits_param = req.params.get('digits')
+
+        if digits_param is not None:
+            try:
+                digits = int(digits_param)
+            except ValueError:
+                digits = 10   # A default
+
+            if digits > 0:
+                digit_string = pi_digits_Python(digits)
+
+                # Insert a decimal point in the return value
+                return func.HttpResponse(digit_string[:1] + '.' + digit_string[1:])
+
+        return func.HttpResponse(
+             "Please pass the URL parameter ?digits= to specify a positive number of digits.",
+             status_code=400
+        )
+    ```
+
+1. Because the code supports only HTTP GET, modify `functions.json` so that the `"methods"` collection contains only `"get"` (that is, remove `"post"`).
+
+1. Start the debugger by pressing F5 or selecting the **Debug** > **Start Debugging** menu command. The output window should now show both endpoints in your project:
+
+    ```
+    Http Functions:
+
+            DigitsOfPi: [GET] http://localhost:7071/api/DigitsOfPi
+
+            HttpExample: [GET,POST] http://localhost:7071/api/HttpExample
+    ```
+
+1. In a browser, or from curl, make a request to `http://localhost:7071/api/DigitsOfPi?digits=125` and observe the output. (You might notice that the code algorithm isn't entirely accurate, but we'll leave the improvements to you!) Stop the debugger when you're finished.
+
+1. Redeploy the code by using the **Deploy to Function App** in the **Azure: Functions** explorer. When prompted, select the Function App created previously.
+
+1. Once deployment finishes (it takes a few minutes!), the output window shows the public endpoints with which you can repeat your tests.
+
+## Clean up resources
+
+The Function App you created includes resources that can incur minimal costs (refer to [Functions Pricing](https://azure.microsoft.com/pricing/details/functions/)). To clean up the resources, right-click the Function App in the **Azure: Functions** explorer and select **Delete Function App**. You can also visit the [Azure portal](https://portal.azure.com), select **Resource groups** from the left-side navigation pane, select the resource group that was created in the process of this tutorial, and then use the **Delete resource group** command.
+
+## Next steps
+
+Congratulations on completing this walkthrough of deploying Python code to Azure Functions! You're now well positioned to create many more serverless functions.
```

Comment:
COPY (style?): Is this a common writing style for the VS Code docs? The tutorials on Azure tend to avoid using kudos like this, making direct appeals to the user, etc.

Reply:
Yes, the VS Code docs are more casual, hence also the tongue-in-cheek remark earlier (which I need to remove anyway for other reasons).